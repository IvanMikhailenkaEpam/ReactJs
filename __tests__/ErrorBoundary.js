import React from 'react';
import { shallow } from 'enzyme';
import ErrorBoundary from "../src/components/error-boundary/ErrorBoundaty";

describe('ErrorBoundary', () => {
  it('renders SearchArea correctly', () => {
    const component = shallow(<ErrorBoundary />);

    expect(component).toMatchSnapshot();
  });
});
