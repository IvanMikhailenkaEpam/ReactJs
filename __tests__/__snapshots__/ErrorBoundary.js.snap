// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary renders SearchArea correctly 1`] = `""`;
